ksqldb-engine/src/main/java/io/confluent/ksql/planner/LogicalPlanner.java

```diff
@@ -143,6 +145,9 @@ public OutputNode buildPlan() {
 
     if (analysis.getRefinementInfo().isPresent()
         && analysis.getRefinementInfo().get().getOutputRefinement() == OutputRefinement.FINAL) {
+      if (!ksqlConfig.getBoolean(KsqlConfig.KSQL_SUPPRESS_ENABLED)) {
+        throw new KsqlException("Suppression is currently disabled in the KsqlConfig.");
```

Comment:
let's just toss in the config name here so that it's easier for people to track it down
```suggestion
        throw new KsqlException("Suppression is currently disabled. You can enable it by setting " + KsqlConfig.KSQL_SUPPRESS_ENABLED + " to true");
```